website/docs/style/design-tokens/examples/custom-component.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import Button from '@semcore/ui/button';
+import Switch from '@semcore/ui/switch';
+import { Box, Flex } from '@semcore/ui/flex-box';
+import { ThemeProvider } from '@semcore/utils/lib/ThemeProvider';
+import "@semcore/utils/style/var.css";
+
+const violetPrimaryButtonTheme = {
+  '--intergalactic-control-primary-info': '#8649e1',
+};
+
+const CustomComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const toggleVisible = () => {
+    setVisible(!visible);
+  };
+
+  const styles = `
+    .popper {
+      box-shadow: var(--intergalactic-box-shadow-popper, 0px 1px 12px 0px rgba(25, 27, 35, 0.15));
+      padding: var(--intergalactic-form-control-m, 28px) var(--intergalactic-form-control-l, 40px);
+    }
+    .kraken {
+      position: absolute;
+      top: 0;
+      left: 80px;
+      display: ${visible ? 'block' : 'none'}
```

Comment:
psl put visible into inline style while `<style />` content move outside of component initialization